.github/workflows/algolia-integrity.yml

```diff
@@ -0,0 +1,56 @@
+# Note: this is a shared pipeline used by other repositories. 
+# Docs: https://docs.github.com/en/actions/using-workflows/reusing-workflows
+
+on:
+  workflow_call:
+    secrets:
+      domain:
+        required: true
+      lockFilePath:
+        required: true
+      docsBaseDir:
+        required: true
+
+jobs:
+  algolia-records-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Fetch
+        run: git fetch origin master
+
+      - name: Setup env
+        uses: the-guild-org/shared-config/setup@main
+        with:
+          nodeVersion: 18
+
+      - name: Algolia generate
+        run: yarn workspace website theguild-nextra-algolia -o ${{ secrets.lockFilePath }} --domain ${{ secrets.domain }} --docsBaseDir ${{ secrets.docsBaseDir }}
+
+      - name: Prettier
+        run: yarn prettier
```

Comment:
Where `prettier` is coming from? is it intended to come from the project as script or as a runtime? 
I can see a potential issue where it fails, do we really need to run it as part of this workflow? (assuming that each project already handles prettier on a separate flow, and also some projects might not have prettier at all)